src/Commands/Scheduled/LinkChecker.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Portable\FilaCms\Commands\Scheduled;
+
+use Illuminate\Console\Command;
+use Portable\FilaCms\FilaCms;
+use Portable\FilaCms\Models\LinkCheck;
+use Str;
+use Portable\FilaCms\Jobs\CheckLink;
+
+class LinkChecker extends Command
+{
+    protected $signature = 'fila-cms:verify-links';
+
+    protected $description = 'Install Fila CMS';
+
+    public function handle()
+    {
+        $models = (new FilaCms())->getRawContentModels();
+        $batch = Str::random(8);
+
+        foreach ($models as $model => $resource) {
+            $modelClass = $model::orderBy('id', 'desc')->chunk(10, function ($records) use ($resource, $batch) {
```

Comment:
`each` might be a better use here as that would remove one of the foreach loops.